api_tester/lib/api_tests/models/store_transaction_api_test.dart

```diff
@@ -8,29 +8,22 @@ class _StoreTransactionApiTest {
     StoreTransaction transaction = StoreTransaction.fromJson(json);
   }
 
-  void _checkToJson(StoreTransaction transaction) {
-    Map<String, dynamic> json = transaction.toJson();
-  }
-
   void _checkConstructor(
       String transactionIdentifier,
-      String revenueCatIdentifier,
       String productIdentifier,
       String purchaseDate) {
     StoreTransaction transaction =
-        StoreTransaction(revenueCatIdentifier, productIdentifier, purchaseDate);
+        StoreTransaction(transactionIdentifier, productIdentifier, purchaseDate);
     StoreTransaction transaction2 =
-        StoreTransaction.create(
```

Comment:
Another breaking change. Seems we had a `create` function for StoreTransaction, but I think it's better to keep the same patter we have everywhere else.